.buildkite/pipeline-release-operatorhub.yml

```diff
@@ -0,0 +1,56 @@
+
+agents:
+  image: docker.elastic.co/ci-agent-images/cloud-k8s-operator/buildkite-agent:latest
+  cpu: "4"
+  memory: "2G"
+
+steps:
+
+  - group: redhat-release
+
+    steps:
+
+      - label: "build operatorhub tool"
+        key: "build-operatorhub-tool"
+        if: build.message == "run operatorhub release"
```

Comment:
Since `pipeline-release-operatorhub.yml` is uploaded only `if: build.message == "run operatorhub release"` dans `pipeline.yml`, it seems redundant to repeat the conditional for all steps.